src/backend/joanie/core/api/client.py

```diff
@@ -813,3 +813,33 @@ def wish(self, request, pk=None):
         else:
             is_wished = models.CourseWish.objects.filter(**params).exists()
         return Response({"status": is_wished})
+
+
+class UserViewSet(mixins.RetrieveModelMixin, viewsets.GenericViewSet):
+    """
+    User ViewSet
+    """
+
+    permission_classes = [permissions.AccessPermission]
+    serializer_class = serializers.UserSerializer
+
+    def get_queryset(self):
+        """
+        Only return users if a query is provided to filter them.
+        """
+        user = self.request.user
+
+        return models.User.objects.get(id=user.id)
+
+    @action(
+        detail=False,
+        methods=["get"],
+        permission_classes=[permissions.IsAuthenticated],
+    )
+    # pylint: disable=invalid-name
+    def me(self, request):
```

Comment:
What do you think about calling the method `me` such as : 
`def get_me(self, request) :`

And add those two parameters (url_name, url_path) in the decorator such as : 
```
    @action(
        detail=False,
        methods=["get"],
        url_name="me",
        url_path="me",
        permission_classes=[permissions.IsAuthenticated],
    )
```